pyglmnet/utils.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
```

Comment:
can you add a minimal docstring, authors and the license to the file? Something like this: https://github.com/nilearn/nilearn/blob/master/nilearn/image/image.py